guides/common/modules/proc_configuring-repositories-deb.adoc

```diff
@@ -20,7 +20,7 @@
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install} /tmp/{PuppetReleaseDeb}
+# {client-package-install-el8} /tmp/{PuppetReleaseDeb}
```

Comment:
This looks strange -- installing a debian package with the el8 package manager?

Reply:
Yeah, fixed line 9 and 23.